Mixpanel.xcodeproj/project.pbxproj

```diff
@@ -1026,6 +939,7 @@
 		E11594861CFF1491007F8B4F /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				APPLICATION_EXTENSION_API_ONLY = YES;
```

Comment:
oh as long as it only affects the appex part of the app i'm okay with either setting for this